### DISCUSSION TITLE
DX , reset password, can we make it simple?

### DISCUSSION THREAD

**Discussion by AHMED-5G** [supabase/supabase#16726] [General]
As I described in issue [React Native - Expo, Password Reset, redirect URL and Exchange authorization code](https://github.com/supabase/supabase/issues/16722) it's complex to reset a password, so can we make it simple ? Like:

Frontend :  hi, this user wants to change his password ?

Backend : ok, tell him to get a 5-digit code from his email, then send it with his email and new password before 60 sec from now.

Ok, thanks.

**Comment by lucksp** [comment]
It would be nice to have a way to reset the password from inside the Mobile application itself, without having to send an email...
